cmd/aws-cloud-controller-manager/main.go

```diff
@@ -63,11 +65,12 @@ func main() {
 		klog.Fatalf("unable to initialize command options: %v", err)
 	}
 
-	controllerInitializers := app.DefaultInitFuncConstructors
+	controllerInitializers := acm.BuildControllerInitializers()
 	fss := cliflag.NamedFlagSets{}
 	command := app.NewCloudControllerManagerCommand(opts, cloudInitializer, controllerInitializers, fss, wait.NeverStop)
 
 	if err := command.Execute(); err != nil {
+		klog.Fatalf("unable to execute command: %v", err)
```

Comment:
`klog.Fatalf()` and `os.Exit()` are redundant: https://github.com/kubernetes/klog/blob/main/klog.go#L1547, you can remove the `os.Exit()` statement.

Reply:
done